blog/_posts/2017-06-30-collections-0.2.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog
+post-type: blog
+by: Julien Richard-Foy
+title: Redesigned collections v0.2.0
+---
+
+We are happy to announce the
+[0.2.0 release](https://index.scala-lang.org/scala/collection-strawman/collection-strawman/0.2.0)
+of the redesigned collections!
+
+You can read more about our effort to redesign the standard collections in this previous
```

Comment:
Summarize what the new collections are about. Don’t point people to read a whole new document. You can make a summary in 3-4 sentences, easy.